konduit-serving-deb/pom.xml

```diff
@@ -14,10 +14,10 @@
     <name>konduit-serving-deb</name>
     <properties>
         <jdeb.signPackage>false</jdeb.signPackage>
-        <prefix>/opt/konduit/serving</prefix>
+        <prefix>/opt/konduit</prefix>
```

Comment:
Why remove the sub folder?

Reply:
For consistency across platforms. The tar and zip packages also contains everything in the konduit root folder